window/src/os/macos/window.rs

```diff
@@ -1149,6 +1180,16 @@ impl WindowInner {
         }
     }
 
+    fn set_level(&mut self, level: i64) {
+        unsafe {
+            NSWindow::setLevel_(*self.window, level);
+        }
+    }
+
+    fn level(&self) -> i64 {
+        unsafe { NSWindow::level(*self.window) }
+    }
+
```

Comment:
According to both:

* https://github.com/rust-windowing/winit/blob/b3c87caa7c040111d8dcae764fe903f54df77426/src/platform_impl/macos/appkit/window.rs#L313
* https://jameshfisher.com/2020/08/03/what-is-the-order-of-nswindow-levels/

The macOS floating concept has level 3 rather than the 22 you selected in here.

Winit chooses to use `-1` as one-below-normal for `AlwaysOnBottom`.

```suggestion
    /// <https://developer.apple.com/documentation/appkit/nswindow/level>
    /// <https://jameshfisher.com/2020/08/03/what-is-the-order-of-nswindow-levels/>
    fn set_level(&mut self, level: WindowLevel) {
        unsafe {
            NSWindow::setLevel_(*self.window, match level {
               WindowLevel::Normal => 0,
               WindowLevel::AlwaysOnTop => 3,
               WindowLevel::AlwaysOnBottom => -1,               
            });
        }
    }
```